docs/authentication/guides/sso/configure-oidc-react-okta.md

```diff
@@ -79,7 +79,7 @@ AUTH_OIDC_CLIENT_ID=your-client-id
 AUTH_OIDC_CLIENT_SECRET=your-client-secret
 AUTH_OIDC_DISCOVERY_URI=https://your-okta-domain.com/.well-known/openid-configuration
 AUTH_OIDC_BASE_URL=your-datahub-url
-AUTH_OIDC_SCOPE="openid profile email groups"
+AUTH_OIDC_SCOPE="openid email profile groups"
```

Comment:
Could you verify one more time that the order actually matters here? I tried reproducing this locally and switching the order made no difference.

Reply:
@aditya-radhakrishnan Yes, I have verified once again order matters